lib/versioning/node/index.ts

```diff
@@ -28,6 +39,11 @@ function getNewValue({
   return res;
 }
 
+export function valueToVersion(value: string): string {
+  const schedule = findScheduleForCodename(value);
+  return schedule?.version || value;
+}
+
 export { isValid };
```

Comment:
I am not sure whether `isValid` needs to be updated to understand codenames. Happy to get input here!

Reply:
Yes, needs to return true for supported codenames.